src/utils/breakpoints.ts

```diff
@@ -20,20 +21,16 @@ export const bpMaxSM = `@media (max-width: ${bp.md - 1}px)`
 export const bpMaxMD = `@media (max-width: ${bp.lg - 1}px)`
 export const bpMaxLG = `@media (max-width: ${bp.xl - 1}px)`
 
-const useBreakpoints = createBreakpoint(bp)
+const getBreakpoint = createBreakpoint(bp)
 
-const useBreakpoint = () => {
-  const breakpoint = useBreakpoints()
-
-  const xs = breakpoint == 'xs'
-  const sm = breakpoint == 'sm' || xs
-  const md = breakpoint == 'md' || sm
-  const lg = breakpoint == 'lg' || md
-  const xl = breakpoint == 'xl' || lg
-  const xl2 = breakpoint == '2xl' || xl
-  const xl3 = breakpoint == '3xl' || xl2
-
-  return {xs, sm, md, lg, xl, xl2, xl3}
+const breakpoints = () => {
+  const breakpoint = getBreakpoint()
+  const keys = Object.keys(bp)
+  const foundInd = keys.indexOf(breakpoint)
+  return keys
+    .map((key, ind) => ({['isMin' + key.toUpperCase()]: ind <= foundInd}))
+    .filter((_, ind) => ind)
+    .reduce((acc, cur) => ({...acc, ...cur}), {})
 }
```

Comment:
Eventually we get the object like:
```
{
  isMin2XL: false
  isMin3XL: false
  isMinLG: true
  isMinMD: true
  isMinSM: true
  isMinXL: true
}
```

only "mobile view" (i.e. `xs`) could be considered like `!isMinSM`

Reply:
![Screen+Recording+2021-01-20+at+06 05 57+PM](https://user-images.githubusercontent.com/1519448/105202068-421f7f00-5b4a-11eb-8fa3-77d4815d3f74.gif)